docker-sync.yml

```diff
@@ -0,0 +1,7 @@
+version: "2"
+
+syncs:
+  #IMPORTANT: ensure this name is unique and does not match your other application container name
+  app_server-sync: #tip: add -sync and you keep consistent names as a convention
```

Comment:
Hey, please add the right `sync_userid` and `sync_groupid`. Also, exclude the unnecessary files e.g. 
`sync_excludes: ['.git', '.idea', 'dev-ops/docker/_volumes', 'var', '.docker-sync']`.

Reply:
You can see an example here: https://github.com/shopware/development/commit/d161be8abd32a348cfdb0d3aaa9831738013d93f